src/Repository/CaseEntityRepository.php

```diff
@@ -18,4 +18,13 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, CaseEntity::class);
     }
+
+    public function findLatestCase(): ?CaseEntity
```

Comment:
I know we talked about how case numbers should not be affected by boards. But in hindsight, should it be affected by municipality?

Reply:
Modified `findLatestCase` to take into account which municipality the case it to be created under, i.e. different counter for each municipality.